release-notes/3.0/preview/api-diff/preview4/3.0-preview4_System.Data.md

```diff
@@ -0,0 +1,100 @@
++        public static DataView AsDataView<T>(this EnumerableRowCollection<T> source) where T : DataRow;
++        public static EnumerableRowCollection<DataRow> AsEnumerable(this DataTable source);
++        public static DataTable CopyToDataTable<T>(this IEnumerable<T> source) where T : DataRow;
++        public static void CopyToDataTable<T>(this IEnumerable<T> source, DataTable table, LoadOption options) where T : DataRow;
++        public static void CopyToDataTable<T>(this IEnumerable<T> source, DataTable table, LoadOption options, FillErrorEventHandler errorHandler) where T : DataRow;
++    }
++    public abstract class EnumerableRowCollection : IEnumerable {
++        IEnumerator System.Collections.IEnumerable.GetEnumerator();
++    }
++    public class EnumerableRowCollection<TRow> : EnumerableRowCollection, IEnumerable, IEnumerable<TRow> {
++        public IEnumerator<TRow> GetEnumerator();
++        IEnumerator System.Collections.IEnumerable.GetEnumerator();
++    }
++    public static class EnumerableRowCollectionExtensions {
++        public static EnumerableRowCollection<TResult> Cast<TResult>(this EnumerableRowCollection source);
++        public static OrderedEnumerableRowCollection<TRow> OrderBy<TRow, TKey>(this EnumerableRowCollection<TRow> source, Func<TRow, TKey> keySelector);
++        public static OrderedEnumerableRowCollection<TRow> OrderBy<TRow, TKey>(this EnumerableRowCollection<TRow> source, Func<TRow, TKey> keySelector, IComparer<TKey> comparer);
++        public static OrderedEnumerableRowCollection<TRow> OrderByDescending<TRow, TKey>(this EnumerableRowCollection<TRow> source, Func<TRow, TKey> keySelector);
++        public static OrderedEnumerableRowCollection<TRow> OrderByDescending<TRow, TKey>(this EnumerableRowCollection<TRow> source, Func<TRow, TKey> keySelector, IComparer<TKey> comparer);
++        public static EnumerableRowCollection<S> Select<TRow, S>(this EnumerableRowCollection<TRow> source, Func<TRow, S> selector);
++        public static OrderedEnumerableRowCollection<TRow> ThenBy<TRow, TKey>(this OrderedEnumerableRowCollection<TRow> source, Func<TRow, TKey> keySelector);
++        public static OrderedEnumerableRowCollection<TRow> ThenBy<TRow, TKey>(this OrderedEnumerableRowCollection<TRow> source, Func<TRow, TKey> keySelector, IComparer<TKey> comparer);
++        public static OrderedEnumerableRowCollection<TRow> ThenByDescending<TRow, TKey>(this OrderedEnumerableRowCollection<TRow> source, Func<TRow, TKey> keySelector);
++        public static OrderedEnumerableRowCollection<TRow> ThenByDescending<TRow, TKey>(this OrderedEnumerableRowCollection<TRow> source, Func<TRow, TKey> keySelector, IComparer<TKey> comparer);
++        public static EnumerableRowCollection<TRow> Where<TRow>(this EnumerableRowCollection<TRow> source, Func<TRow, bool> predicate);
++    }
++    public sealed class OrderedEnumerableRowCollection<TRow> : EnumerableRowCollection<TRow>
```

Comment:
cc @ericstj  to verify

Reply:
I defer to @divega. I just helped do the work.

Reply:
The rest of the System.Data APIs LGTM.